### PR TITLE
metrics,server: add a get_token_duration_seconds metrics

### PR DESCRIPTION
**Pull request by tiancaiamao**
## What have you changed? (mandatory)

Add a metrics `get_token_duration_seconds`, if concurrent limit is reached, we should be able see blocking on get token.

image

## What is the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No

## Does this PR affect tidb-ansible update? (mandatory)

Later I'll update tidb-ansible grafana script.

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: server/server.go [hunk 1]
```diff
@@ -102,7 +102,10 @@ func (s *Server) ConnectionCount() int {
 }
 
 func (s *Server) getToken() *Token {
-	return s.concurrentLimiter.Get()
+	start := time.Now()
+	tok := s.concurrentLimiter.Get()
+	metrics.GetTokenDurationHistogram.Observe(time.Since(start).Seconds())
```
Comment (ngaut) [inline server/server.go]: Could you use million seconds?

**Comment by tiancaiamao** [comment]
PTAL @ngaut

**Pull request merged**